mappings/net/minecraft/world/gen/DensityFunctions.mapping

```diff
@@ -0,0 +1,355 @@
+CLASS net/minecraft/unmapped/C_mdkmcezi net/minecraft/world/gen/DensityFunctions
+	METHOD m_ajxltzce blendOffset ()Lnet/minecraft/unmapped/C_kzwapyud;
+	METHOD m_akixioyj blendDensity (Lnet/minecraft/unmapped/C_kzwapyud;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 input
+	METHOD m_bavhhzle terrainShaperSpline (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_oqsszdfo;DD)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 continentalness
+		ARG 1 erosion
+		ARG 2 weirdness
+		ARG 3 spline
+		ARG 4 minValue
+		ARG 6 maxValue
+	METHOD m_diiubzxc shiftedNoise2d (Lnet/minecraft/unmapped/C_kzwapyud;Lnet/minecraft/unmapped/C_kzwapyud;DLnet/minecraft/unmapped/C_rxrtfjrt;)Lnet/minecraft/unmapped/C_kzwapyud;
+		ARG 0 shiftX
+		ARG 1 shiftZ
+		ARG 2 xzScale
```

Comment:
How about `horizontalScale`? Same thing for `yScale`, could be `verticalScale`

Reply:
I'd agree